src/content-chunks/android-sdk-api-intro-text.mdx

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Since these are no longer used, could you remove all the stuff in `src/content-chunks`?

Reply:
yep, pushed up a little too quickly. updated it and pushed it up